pipeline-scripts/buildlib.groovy

```diff
@@ -1254,4 +1281,35 @@ def assertBuildPermitted(doozerOpts) {
     }
 }
 
+/**
+ * Run elliott find-builds and attach to given advisory.
+ * It looks for builds twice (rhel-7 and rhel-8) for OCP 4.y
+ * Side-effect: Advisory states are changed to "NEW FILES" in order to attach builds.
+ *
+ * @param String[] kinds: List of build kinds you want to find (e.g. ["rpm", "image"])
+ * @param String buildVersion: OCP build version (e.g. 4.2, 4.1, 3.11)
+ */
+def attachBuildsToAdvisory(String[] kinds, String buildVersion) {
+    def groupOpt = "-g openshift-${buildVersion}"
+    def isOCP4 = buildVersion.startsWith("4.")
+    def rhel8branchOpt = "--branch rhaos-${buildVersion}-rhel-8"
+
+    try {
+        if ("rpm" in kinds) {
+            elliott("${groupOpt} change-stage -s NEW_FILES --use-default-advisory rpm")
+            elliott("${groupOpt} find-builds -k rpm --use-default-advisory rpm")
+            if (isOCP4) {
+                elliott("${groupOpt} find-builds -k rpm --use-default-advisory rpm ${rhel8branchOpt}")
```

Comment:
This is to attach RHEL 8 targeted RPMs. But I don't see the code to attach RHEL 8 based images.

Reply:
@vfreex I thought `find-builds -k image` just finds all builds, RHEL7 and 8 ... isn't that the case? I remember @shiywang working on it, for the `--between` flag

Reply:
It is still need `--branch` to get those images attached to proper product versions. When Shiyang's PR is finished, we probably don't need  to call twice. But I didn't test it. @shiywang Could you also confirm this?

Reply:
@vfreex that looks to be a problem only with the `--between` flag. Normally, of course, that's what we want in 4.y. But in this case, dumping them all into the one advisory works.
I tested it :) https://saml.buildvm.openshift.eng.bos.redhat.com:8888/job/hack/job/lmeyer/job/lmeyer-dev/job/dev-build%252Fsweep/13/
(there are a few other minor issues to address though as it turns out)

Reply:
Actually I'm just a fool and you're right. It doesn't pick up the rhel8 image builds this way. And it doesn't work if I use `--branch el8` either.
https://saml.buildvm.openshift.eng.bos.redhat.com:8888/job/hack/job/lmeyer/job/lmeyer-dev/job/dev-build%252Fsweep/14/console
Well, let's wait on elliott to work the way we expect.